lib/plugin.js

```diff
@@ -115,7 +115,13 @@ internals.schema.strategy = Joi.object({
         timeSkewSec: Joi.number().integer().min(0).default(0)
     })
         .when('.validate', { is: Joi.not(false), then: Joi.allow(false) })
-        .required()
+        .required(),
+
+    headerName: Joi.alternatives()
+        .try(Joi.string(), Joi.boolean()).default('authorization').optional(),
+
+    cookieName: Joi.alternatives()
+        .try(Joi.string(), Joi.boolean()).default('token').optional()
```

Comment:
Ideally the string would be restricted to valid cookie names using an appropriate regex.

Reply:
But doesn't that reduce flexibility. Also the issue author and also a lot of other people actually have a requirement to use the names that they need. Like for example if they have a refresh token and an auth token then it makes sense for them to have two cookies in which the authentication one might be better named as `auth_token`

Reply:
I'm talking about verifying that the string value is valid to include as a cookie name, eg. no spaces, etc.

Reply:
This is now added, now only valid cookie names and header names are accepted